packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -879,22 +884,22 @@ export class PipelineEditor extends React.Component<
    *
    * @returns true if the pipeline is invalid.
    */
-  invalidPipeline(): boolean {
-    let invalidPipeline = false;
+  validateAllNodes(): boolean {
+    let invalidPipeline = true;
```

Comment:
given you inverted the meaning of this value the var name should probably be updated as well (same with the jsdoc above)